controllers/front/CartController.php

```diff
@@ -135,19 +135,13 @@ public function displayAjaxUpdate()
         $productQuantity = $updatedProduct['quantity'] ?? 0;
 
         if (!$this->errors) {
-            $presentedCart = $this->cart_presenter->present($this->context->cart, true);
```

Comment:
I thought we discussed about the possibility to set an optional product filtered here, so that the API correctly returns only whitelisted fields of products (only in this use case)

Reply:
I'll do it in the next pull request, that will implement php attributes and add an attribute to allow overriding value in the array manner

but if you want I can, just for the time between this PR and the next one, put the intermediary class that allows editing, I did it earlier but I removed the commit